GliaWidgets/Sources/ViewModel/Call/CallViewModel.swift

```diff
@@ -367,7 +367,11 @@ extension CallViewModel {
     private func showLocalVideo(with stream: CoreSdkClient.VideoStreamable) {
         action?(.switchToVideoMode)
         action?(.setLocalVideo(stream.getStreamView()))
-        stream.playVideo()
+        stream.onHold = { isOnHold in
+            if !isOnHold {
+                stream.playVideo()
+            }
```

Comment:
perhaps I missed something, however, I see that we `play` stream, but don't pause it, should we do it?

Reply:
No, because `pause` and `resume` are called when onHold is triggered. But with this situation, video stream was added when visitor was on hold , not taking into account that during on hold we are not allowed to show the video. If operator resumes the call, so does the SDK, because that logic works correctly. And, if operator puts the visitor on hold again, that is executed correctly too

Reply:
then this closure looks a little bit strange in that we handle only one case but closure provides two possible cases. perhaps we can rethink & rework it later. 
or leave comment for now that it is expected that we ignore `on hold` case.